multimodal/health_multimodal/text/data/io.py

```diff
@@ -0,0 +1,47 @@
+#  -------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  -------------------------------------------------------------------------------------------
+
+import logging
+from typing import Any, List, Union
+
+from transformers import BertTokenizer
+
+
+class TextInput:
+    """
+    Text input class that can be used for inference and deployment.
+    Implements tokenizer related operations and ensure that input strings
+    conform with the standards expected from a BERT model.
+
+    :param tokenizer: A BertTokenizer object.
+    """
+
+    def __init__(self, tokenizer: BertTokenizer) -> None:
+        self.tokenizer = tokenizer
+
+    def tokenize_input_prompts(self, prompts: Union[str, List[str]], verbose: bool) -> Any:
```

Comment:
Here it looks like we expect `prompts` to be single sentences, should we write this down in the docs or add some check on this? Since this function does not place [SEP] tokens between sentences.

Reply:
Yes, there should be a docstring.

Reply:
Actually, after reading again the function: what do you mean by "single sentences"?

Reply:
@Shruthi42, can you please work on this one?

Reply:
A single `str` is a single sentence here. I've added a docstring for this method to explain the input.